package.json

```diff
@@ -26,6 +26,7 @@
   "dependencies": {
     "coffee-script": "~1.12.2",
     "conventional-changelog-cli": "~1.3.1",
+    "cross-spawn": "^5.1.0",
```

Comment:
Sorry I missed this comment, would you like me to send a PR with the `package.json` update or leave it for now?